Cargo.toml

```diff
@@ -19,29 +25,29 @@ once_cell = { version = "1.17.1", default-features = false }
 thiserror-no-std = { version = "2.0.2", default-features = false }
 cairo-lang-casm-contract-class = { git = "https://github.com/keep-starknet-strange/cairo", branch = "no_std-support-8bbf530", default-features = false }
 primitive-types = { version = "0.12.1", default-features = false, features = [
-    "serde_no_std",
+  "serde_no_std",
 ] }
 hex = { version = "0.4.3", default-features = false, features = ["alloc"] }
 indexmap = { git = "https://github.com/bluss/indexmap", rev = "ca5f848e10c31e80aeaad0720d14aa2f6dd6cfb1", features = [
-    "serde",
+  "serde",
 ], default-features = false }
 serde = { version = "1.0.130", features = [
-    "derive",
-    "rc",
+  "derive",
+  "rc",
 ], default-features = false }
 serde_json = { version = "1.0.81", default-features = false, features = [
-    "alloc",
+  "alloc",
 ] }
-starknet-crypto = { version = "0.5.1", default-features = false }
+starknet-crypto = { git = "https://github.com/xJonathanLEI/starknet-rs.git", rev = "a35ce22", default-features = false }
```

Comment:
There was no release? I though Jonathan told me it had released those. I would prefer use the release rather than a gh hash

Reply:
I don't think so, latest release for `starknet-crypto` is one month ago
https://crates.io/crates/starknet-crypto/versions

Reply:
kay